test/plugins/scenarios/microv4/config/excepted.yml

```diff
@@ -0,0 +1,85 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+segmentItems:
+  - serviceName: micro-server
+    segmentSize: ge 1
+    segments:
+      - segmentId: not null
+        spans:
+          - operationName: go.micro.srv.greeter.Say.Hello
```

Comment:
I think the only concern of this implementation would be, this would never be an `endpoint`, and not in the `endpoint dependency` chain as well.

Reply:
In this particular case, I think it falls into one scenario we discussed. Whether span should a wrapper to update cross-goroutine. Such as here, should we consider no microv4 span, but HTTP server span would be updated later.